projects/go-lib/src/lib/components/go-datepicker/go-calendar.component.ts

```diff
@@ -63,15 +70,11 @@ export class GoCalendarComponent implements OnDestroy, OnInit {
   }
 
   ngOnInit(): void {
-    this.subscription = this.calendar.calendarOpen.subscribe((value: boolean) => {
-      if (value) {
         this.selectedDate = this.calendar.selectedDate;
         this.initializeDate();
-      }
-      this.opened = value;
-    });
-
-    this.dateAdapter.setLocale(this.locale);
+        this.appendToCalender();
+        this.opened = true;
```

Comment:
We can actually remove the `opened` property altogether because we're checking the opened status with the `ngIf` on the `go-calendar` component itself in the datepicker component.